### PR TITLE
Yemad 10 quiz results

### PR DESCRIPTION
**Pull request by youssef-emad**
@fakhreldin-m-abdelkhalik 

review and merge

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -28,6 +28,8 @@
 
     post 'signin', to: 'sessions#instructor_create'
 
+    get '/quiz/:quiz_id/group/:group_id', to: 'quizzes#group_result'
```
Comment (fakhreldin-m-abdelkhalik) [inline config/routes.rb]: Change the route to be quizzes instead of quiz @Youssef-Emad  to follow one convention for quizzes routes .

**Comment by youssef-emad** [comment]
@fakhreldin-m-abdelkhalik 
ok no problem :smile:

**Comment by youssef-emad** [comment]
@fakhreldin-m-abdelkhalik 
DONE

**Comment by fakhreldin-m-abdelkhalik** [comment]
Well Done @Youssef-Emad

**Pull request merged**